test/integration/discovery/discovery_helpers.bash

```diff
@@ -10,6 +10,11 @@ function discovery_check_swarm_info() {
 	docker_swarm info | grep -q "Nodes: $count"
 }
 
+# Returns true if swarm info outputs is empty (0 nodes).
```

Comment:
Actually, I changed the above (`discovery_check_swarm_info`) to take a parameter so you could do this with:

`discovery_check_swarm_info "$DISCOVERY" 0`